web/sites/default/config/rabbit_hole.behavior_settings.media_type_document.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 1ba8856f-fe58-4231-ba80-497147ff0dba
+langcode: en
+status: true
+dependencies:
+  config:
+    - media.type.document
+id: media_type_document
+action: page_redirect
```

Comment:
Did we figure out the way we were going to test whether this redirect breaks analytics? I'd rather we not break analytics first then figure out a way to fix it 😆